compiler/rustc_mir_transform/src/inline/cycle.rs

```diff
@@ -84,7 +84,8 @@ crate fn mir_callgraph_reachable(
                 | InstanceDef::ReifyShim(_)
                 | InstanceDef::FnPtrShim(..)
                 | InstanceDef::ClosureOnceShim { .. }
-                | InstanceDef::CloneShim(..) => {}
+                | InstanceDef::CloneShim(..)
+                | InstanceDef::AsRepr(..) => {}
```

Comment:
I think this one can be `continue`. The `AsRepr` MIR never contains a call, right?